### PR TITLE
Add Duo Security format

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: Duo/duo-security/_meta/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+uuid: 547234b3-82ea-4507-b28f-3ee3cd5b9a8e
+name: Duo Security [BETA]
+slug: duo-security
+
+description: >-
+  Duo Security offers solutions for strong authentication. This intake ingest accounting and administration logs.
+
+data_sources:
+  Authentication logs: Duo Secirty provides audit logs about authentication sessions
```
Comment (squioc) [inline Duo/duo-security/_meta/manifest.yml:9]:
```suggestion
  Authentication logs: Duo Security provides audit logs about authentication sessions
```

File: Duo/_meta/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+uuid: be246024-5e6b-4df0-83dc-bd61226fbffd
+name: Duo [BETA]
```
Comment (squioc) [inline Duo/_meta/manifest.yml:2]: Beta flag is not required for module name
```suggestion
name: Duo
```

File: Duo/duo-security/tests/test_auth_log.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "event": {
+      "kind": "event",
+      "category": ["authentication"],
+      "type": ["info"],
+      "dataset": "auth_log"
+    },
+    "@timestamp": "2020-02-13T18:56:20.351346Z",
+    "user": {
+      "email": "narroway@example.com",
+      "id": "DU3KC77WJ06Y5HIV7XKQ"
```
Comment (squioc) [inline Duo/duo-security/tests/test_auth_log.json:22]: I wonder if we can fill ECS field `user.roles` with the content of `user.groups`?

**Review by squioc — CHANGES_REQUESTED** [review]
Nice job.

File: Duo/duo-security/ingest/parser.yml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+stages:
+  set_timestamp:
+    actions:
+      - set:
+          "@timestamp": "{{ parse_date.date }}"
+
+  set_event_data:
+    actions:
+      - set:
+          event.kind: "event"
```
Comment (squioc) [inline Duo/duo-security/ingest/parser.yml:41]:
```suggestion
          event.kind: "event"
          observer.vendor: "Duo"
          observer.product: "Duo Security"
```

File: Duo/duo-security/tests/test_auth_log.json (hunk 3 above)
Comment (lvoloshyn-sekoia) [reply Duo/duo-security/tests/test_auth_log.json:22]: Make sense, will add it

**Review by squioc — APPROVED** [review]
LGTM

**Review by squioc — APPROVED** [review]

**Pull request merged**